tracer/test/test-applications/integrations/Samples.InstrumentedTests/Vulnerabilities/Json/Newtonsoft.Json/ParseTests.cs

```diff
@@ -0,0 +1,115 @@
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace Samples.InstrumentedTests.Iast.Vulnerabilities.Json.Newtonsoft.Json;
+
+public class ParseTests : InstrumentationTestsBase
+{
+    private readonly string _taintedJson = "{ \"key\": \"value\" }";
+    private readonly string _taintedJsonObjectWithArray = "{\"key\": [\"value1\", \"value2\", {\"key2\": \"value\"}]}";
+    private readonly string _taintedJsonArray = "[\"value1\", \"value2\"]";
+    private readonly string _taintedJsonDifferentTypes = "{ \"name\": \"Chris\", \"age\": 23, \"address\": { \"city\": \"New York\", \"country\": \"America\" }, \"friends\": [ { \"name\": \"Emily\", \"hobbies\": [ \"biking\", \"music\", \"gaming\" ] }, { \"name\": \"John\", \"hobbies\": [ \"soccer\", \"gaming\" ] }, [ \"aString\", { \"obj\": \"val\" } ] ] }";
```

Comment:
FYI, newtonsoft supports a range of other features you should probably test against, e.g.

- https://www.newtonsoft.com/json/help/html/preservereferenceshandlingobject.htm
- https://www.newtonsoft.com/json/help/html/SerializeTypeNameHandling.htm

Also, this doesn't seem to support `JsonConvert.Deserialize<Object>` and associated methods, are they out of scope? 🤔

Reply:
At the moment, this PR only cover the Parse method, the JsonConvert will be supported in another PR.